composer.json

```diff
@@ -17,7 +17,8 @@
         }
     ],
     "require": {
-        "php" : "^7.1 || ^8.0"
+        "php" : "^7.1 || ^8.0",
+        "symfony/polyfill-php80": "^1.17"
```

Comment:
why do we need this polyfil?

Reply:
Because I use the `preg_last_error_msg` function.

Reply:
I don't think these polyfill dependencies really matter. They're fine to have IMO. If we REALLY wanted to avoid it, we could copy-paste the function we are using into the codebase.

Reply:
no it's ok...